conftest.py

```diff
@@ -0,0 +1,18 @@
+from itertools import product
+
+import pytest
+
+from utils import make_col, make_frame
+
+n = [100, 10000]
```

Comment:
I'm not sure I like encoding these sizes globally like this. It could easily become misleading later if not all fixtures etc use it. I'm leaning towards preferring to just include the values in the definition of the fixtures below. What do you think? At minimum we need a comment indicating what `n` is used for.

Reply:
Also, how do we want to separate code between conftest and `utils`?

Reply:
Agreed with the first point. Maybe we should explain the parameters in the docstring of the fixtures.

I'd say helper functions that's involved in data creation should live in conftest. But not sure if we want to migrate rn because there are benchmarks that uses custom parameters adhocly. Unifying all parametrization into conftest is a separate effort to this PR IMO.

Reply:
Also, I think there are more to discuss in that effort. This [thread](https://stackoverflow.com/questions/34466027/in-pytest-what-is-the-use-of-conftest-py-files) mentions that if fixtures defined in conftest not shared by all tests, it may slow down the whole test process. So to what extent do we want to create the test data up front and the re-usability of the fixtures is to be seen.

Reply:
We definitely don't want to end up in a situation like cudf's current tests where test collection is slow because of how many objects are created up front. I think we should prefer fixtures or parametrizing tests using scalar values that can be passed to constructors of cudf objects rather than directly parametrizing with cudf objects. But I agree, we can deal with this in a future PR.